.gitlab-ci.yml

```diff
@@ -355,25 +355,29 @@ doc:refman:deploy:
     variables:
       - $DOCUMENTATION_DEPLOY_KEY
   dependencies:
+    - doc:ml-api:odoc
+    - doc:refman:dune
     - doc:refman
```

Comment:
That seems suspicious, why the double dependency on the `refman`?

Reply:
Is that because of the lack of `coqdoc` build for the stdlib? If so please open a tracker issue so we are reminded to do so and update the deploy rule.

The plan for coqdoc is to have their own built-in rules for libraries so that requires the native Dune support being worked on, but in fact we may use coq2html instead.

Reply:
You are right: the refman is taken from the dune build; the stdlib from the non-dune build.

When the dune build-system will be able to generate the stdlib documentation, I guess that the `doc:refman` job will be removed. That will already mandate to update this job. Do we need an additional issue?

Reply:
I have opened #9649 so I think that'll work.

Reply:
>You are right: the refman is taken from the dune build; the stdlib from the non-dune build.

Stdlib is from build:base though. Or you could go with just the non-dune built refman no?